backend/dataall/modules/metadata_forms/__init__.py

```diff
@@ -0,0 +1,13 @@
+from dataall.base.loader import ModuleInterface, ImportMode
+
+
+class MetadataFormsApiModuleInterface(ModuleInterface):
+    """Implements ModuleInterface for Metadata Forms GraphQl lambda"""
+
+    @classmethod
+    def is_supported(cls, modes):
+        return ImportMode.API in modes
+
+    def __init__(self):
+        import dataall.modules.metadata_forms.api
+        import dataall.modules.metadata_forms.db.enums
```

Comment:
Just confirming where we should be storing `enums`? In `api/` or `services/` or `db/`? Seems like most moved to `services/` as of https://github.com/data-dot-all/dataall/pull/978

I am not strongly opinionated or a particualr format - just want to ensure standardized across

@dlpzx I know you had mentioned this in another PR recently - did you have strong views on this?

Reply:
I need to check, if we can store them in services in this particular case. I'm really afraid of circular dependencies

Reply:
Here we can't move it to Services. Enums are used in repository and models (for polymorphism). So, It would cause circular dependencies